src/app/home/testimonials/testimonials.component.scss

```diff
@@ -0,0 +1,28 @@
+.student-picture {
+  text-align: center;
+  min-height: 40vh;
+  display: flex;
+  align-items: center;
+
+  @media screen and (max-width: 993px) {
+    min-height: auto;
+    display: inherit;
+  }
+
+  img {
+    width: 65%;
```

Comment:
not sure if this widths are actually needed, i will say that we should put the image inside a column definition and let it grown over that column with a minimun/maximun height. Let me know what do you think